int64 iden
---
int64 iden
float64 length
float64 time
float64[] xs
float64[] deltas
